addons/website_forum/models/forum.py

```diff
@@ -592,8 +592,10 @@ def write(self, vals):
                 # update karma except for self-acceptance
                 mult = 1 if vals['is_correct'] else -1
                 if vals['is_correct'] != post.is_correct and post.create_uid.id != self._uid:
-                    post.create_uid.sudo().add_karma(post.forum_id.karma_gen_answer_accepted * mult)
-                    self.env.user.sudo().add_karma(post.forum_id.karma_gen_answer_accept * mult)
+                    post.create_uid.sudo()._add_karma(post.forum_id.karma_gen_answer_accepted * mult, post,
+                                                      _('Answer accepted') if mult > 0 else _('Accepted answer removed'))
```

Comment:
User answer accepted ? (to try to be clearer about accepted other people answer versus its answer is accepted)